test/e2e/tests/permissions.spec.js

```diff
@@ -62,7 +62,7 @@ describe('Permissions', function () {
           text: 'Connected sites',
           tag: 'h2',
         });
-
+        await driver.delay(xLargeDelayMs);
```

Comment:
Is this related in some way?

Reply:
The test started failing without that delay, it was not related to token detection changes.